modules/cnf-running-the-performance-creator-profile.adoc

```diff
@@ -90,7 +90,7 @@ Using this information you can set appropriate values for some of the arguments
 +
 [source,terminal]
 ----
-$ podman run --entrypoint performance-profile-creator -v /must-gather:/must-gather:z registry.redhat.io/openshift4/ose-cluster-node-tuning-operator:v4.11 --info log --must-gather-dir-path /must-gather
+$ podman run --entrypoint performance-profile-creator -v `pwd`/must-gather:/must-gather:z registry.redhat.io/openshift4/ose-cluster-node-tuning-operator:v4.11 --info log --must-gather-dir-path /must-gather
```

Comment:
+1 for the general idea. I think `$( pwd )` is more readable than backticks (``) but I'll defer the final call to doc reviewers